system/Test/Mock/MockCLIConfig.php

```diff
@@ -11,6 +11,7 @@ class MockCLIConfig extends \Config\App
 	public $cookiePath     = '/';
 	public $cookieSecure   = false;
 	public $cookieHTTPOnly = false;
+	public $cookieSameSite = '';
```

Comment:
Just for consistency
```suggestion
	public $cookieSameSite = 'Lax';
```